GOVERNANCE.md

```diff
@@ -653,6 +653,9 @@ Team:
 In addition, Council members are given administrative rights to all repositories within
 the [pymc-devs](https://github.com/pymc-devs) organization.
 
+##### Communication Focused Repositroes
+Some repositories on Github may be used primarily for internal knowledge store and communication, rather than as codebases that are packaged and deployd. In this case permissions, such as write permissions, will be granted in a manner more like other communication platforms such as slack.
```

Comment:
I would change the `rather than as codebases...` to something like `rather than content that is curated and published/released _by the project_`. 

Then also the next sentence: 

```The permissions of such repositories will be set in order to allow the same participation and access levels we use on private project communication channels like Slack. Therefore, similarly to slack, these repositories will be private and write permissions will be given to all recurrent contributors (that is, anyone with access to slack).```